providers/capv/standard/capv_suite_test.go

```diff
@@ -0,0 +1,90 @@
+package standard
+
+import (
+	"context"
+	"path"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	cr "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/clustertest"
+	"github.com/giantswarm/clustertest/pkg/application"
+	"github.com/giantswarm/clustertest/pkg/logger"
+	"github.com/giantswarm/clustertest/pkg/organization"
+	"github.com/giantswarm/clustertest/pkg/utils"
+	"github.com/giantswarm/clustertest/pkg/wait"
+
+	"github.com/giantswarm/cluster-test-suites/common"
+)
+
+const (
+	KubeContext       = "capv"
+	clusterAppVersion = "0.3.1-689cef7898e90f8184ac526369dddf119d9a0af2"
+	defaultAppVersion = "0.8.0-5b825f54f13188ea0c024b3b757e1c253a80924c"
```

Comment:
These shouldn't be hardcoded. These should be defaulted to not being provided so that the latest released version is automatically pulled. `clustertest` then supports the `E2E_OVERRIDE_VERSIONS` environment variable that is used by the E2E tests to override the version being used to match that of the PRs  to test those changes.

Reply:
got ya, I'll check if we need to release them or the latest released versions are fine. Then the workflow is triggered by that gh comment on that corresponding repo and PR, right? I can try that..

Reply:
Correct. Assuming the webhooks are set up which I’m fairly sure I did.